ctapipe/io/hdf5tableio.py

```diff
@@ -440,7 +454,7 @@ def read(self, table_name: str, container: Container):
 
 
 def tr_convert_and_strip_unit(quantity, unit):
-    return quantity.to(unit).value
+    return quantity.to_value(unit)
```

Comment:
This function has no added value anymore, wouldn't it be better to just replace `tr_convert_and_strip_unit` everywhere by `quantity.to_value(unit)`?

Reply:
No, that's not possible. This function is used for the column transforms. 
And that must be callable as `func(col)`. Or you need to introduce a special case for quantities. But I'd rather have a very simple function than a special case in the application of the transforms.

Reply:
You could make it a lambda like for the case where no unit is given, but hey, it doesn't harm anyone. And if you inspect the column transforms, it is readable as opposed to `lambda at <hex adress>`